catalog/task/create-pyxis-image/0.3/create-pyxis-image.yaml

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
I am not seeing how this yaml is different than v 0.2. Maybe I am just missing it (one con of the version structure... the diff is hard to notice). What is it?

Reply:
> one con of the version structure... the diff is hard to notice

Yeah, it would be a good idea to provide two commits for changes like these - one that will just copy the previous version and the other that will make the changes. That way you can easily review the changes without checking out locally and using `diff -r`. Which is the only option right now.

Reply:
But all I see is a change in the parameter description:
```diff
diff -r 0.2 0.3
diff --color -r 0.2/README.md 0.3/README.md
20a21,24
> ## Changes since 0.2
>
> * Use tag present in mapping file if present.
>
diff --color -r 0.2/create-pyxis-image.yaml 0.3/create-pyxis-image.yaml
7c7
<     app.kubernetes.io/version: "0.1"
---
>     app.kubernetes.io/version: "0.3"
28c28
<       description: The tag to use when pushing the container image metadata to Pyxis
---
>       description: Default tag to use if mapping entry does not contain a tag
diff --color -r 0.2/samples/sample_create-pyxis-image_TaskRun.yaml 0.3/samples/sample_create-pyxis-image_TaskRun.yaml
16c16
<     bundle: quay.io/hacbs-release/task-create-pyxis-image:0.2
---
>     bundle: quay.io/hacbs-release/task-create-pyxis-image:0.3
diff --color -r 0.2/tests/run.yaml 0.3/tests/run.yaml
16c16
<     bundle: quay.io/hacbs-release/task-create-pyxis-image:0.2
---
>     bundle: quay.io/hacbs-release/task-create-pyxis-image:0.3
```

@scoheb is that intentional? Do we even need a new version if just the description is changed?

Reply:
> Yeah, it would be a good idea to provide two commits for changes like these 

I agree. good idea.

Reply:
The version was wrong. just figured it was safer to create new version.

Reply:
Oh I see. Then it's fine with me, thanks.

Reply:
I think we should come to some guidelines as a team about when to create new versions. For example, 0.2 was created fixing a syntactical bug in 0.1 (nice catch by Scott, thanks for fixing it). In this case, I think we should have just fixed 0.1 instead of creating 0.2, because 0.1 doesn't work (and will never, due to syntax bug). But, we haven't discussed when to create new versions and when not to